### ISSUE TITLE
Can't use custom pagination template in server side mode.

### ISSUE THREAD

**Issue by effectidea** [Zizzamia/ng-tasty#86]
Hello!
I want to use custom pagination template in server mode. According to docs I set new attribute in 'tasty-pagination' div:
`bind-template-url="folder_in_the_server/template_new.html"`
But I have strange internal angular error in browser console. After debug I see some interesting things: for isolated scope "templateUrl" value is NaN.

Please, can you add to docs some examples with custom pagination template using? I guess, I have some mistakes in my code and can't reproduce right way.
Thank You Very Much!

**Comment by Zizzamia** [comment]
ThankYou to reporting this case, I will make sure to cover your case in the documentation.

**Comment by Zizzamia** [comment]
Hi @effectidea,
here you can find an example with custom template-url http://plnkr.co/edit/0oRWyWMYY9zoQEeNlK7h?p=preview

The problem was in the use of `bind-`, so I made the table directive working with two differente way of passing variable.
1) bind-template-url : all the time you use `bind-` you will pass a `$scope` variable
2) template-url : you will just pas a plain text

So in your case you need use just `template-url`, I tried in the demo, but looks there in a bug in that case.

ThankYou you just help me discover a bug :)

In the end, as you see in the demo I post it in the comment, you just need write this
```html
<div tasty-pagination bind-template-url="templateUrl"></div>
```
```js
$scope.templateUrl = 'localTemplatePagination.html';
```

**Comment by effectidea** [comment]
Thank You, @Zizzamia ! I understood your answer and made changes in my code. Now all works fine!
Best regards!

**Comment by Zizzamia** [comment]
By the way the bug you found it, I just fix it and deploy a new version 0.4.6 of ng-tasty :)
